src/styles/link.css

```diff
@@ -0,0 +1,26 @@
+/* #region link */
+.link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  box-sizing: border-box;
+  height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Use padding instead